stories/dates/date-range-input.js

```diff
@@ -391,6 +391,14 @@ stories.add('custom css variables', () => ({
 
 
 stories.add('compact mode', () => ({
+  props: {
+    hidePlaceholder: {
+      default: boolean('Hide placeholder', true),
+    },
+    coloredBackground: {
+      default: boolean('Colored background', true),
+    },
+  },
```

Comment:
Note for future PRs: the knobs addons are deprecated in favor of controls
https://github.com/storybookjs/storybook/discussions/15060
In next PRs, if you have the opportunity, don't hesitate to use the new system :)